test/LiquidityManager/invest.test.ts

```diff
@@ -62,15 +74,34 @@ describe.only('LiquidityManager#invest', () => {
         )
     }
 
-    async function sortTokens(tokenA: TestERC20, tokenB: TestERC20) {
-        const [addressTokenA, addressTokenB] = await Promise.all([
-            unwrapAddressLike(tokenA),
-            unwrapAddressLike(tokenB),
+    async function sortTokensAndPrices(
+        token0: TestERC20,
+        token0Price: bigint,
+        token1: TestERC20,
+        token1Price: bigint,
```

Comment:
when tokens aren't sorted, we should call them tokenA and tokenB, this way we can be sure about whether they are sorted or not

Reply:
I believe we are also going to use this function in the frontend